Daytime-Operations/Main-Telescope/MT-Calibrations/ComCam-Combined-Calibrations-Procedure/comcam-combined-calibrations-procedure.rst

```diff
@@ -223,11 +277,11 @@ The certified combined calibrations will be available via the collection specifi
     ptc = butler.get('ptc', detector=detector[0], exposure=exposure[0], instrument='LSSTComCam')
 
 
-If ``do_gain_from_flat_pair`` is ``True``, the estimated gains (as well as the measured empirical readout noise from the overscan during Instrument Signature Removal) can be found by requesting the ``cpCovariances`` data structure. In this case, the exposure ID should be one of the two flats used to estimate the gain:
+If ``do_gain_from_flat_pair`` is ``True``, the estimated gains (as well as the measured empirical readout noise from the overscan during Instrument Signature Removal) can be found by requesting the ``cpPtcExtract`` data structure. In this case, the exposure ID should be one of the two flats used to estimate the gain:
 
 .. code-block:: python
 
-    cpCovs = butler.get('cpCovariances', detector=detector[0], exposure=flat1ID, instrument='LSSTComCam')
+    cpCovs = butler.get('cpPtcExtract', detector=detector[0], exposure=flat1ID, instrument='LSSTComCam')
```

Comment:
Just to check (and it applies to the LATISS case as well) that `detector` here is a list of detectors or a camera?  Indexing a detector object like that should return an amplifier.  Looking at the full file, you state that "detectors: Detector IDs that will be pased to the pipeline tasks, given as an array of integers, e.g., [0,1,2,3]."  If these are the same concept, I think you need to change all of the `detector` to `detectors` to make it clear that this is indexing that array of integers.